backend/src/Service/FishbowlService.php

```diff
@@ -45,27 +45,14 @@
  */
 class FishbowlService
 {
-    protected FishbowlRepository $fishbowlRepository;
-    protected RequestStack $requestStack;
-    protected Security $security;
-    protected GuestRepository $guestRepository;
-    protected ParticipantRepository $participantRepository;
-    protected TranslatorInterface $translator;
-
     public function __construct(
-        FishbowlRepository $fishbowlRepository,
-        RequestStack $requestStack,
-        Security $security,
-        GuestRepository $guestRepository,
-        ParticipantRepository $participantRepository,
-        TranslatorInterface $translator
+        readonly private FishbowlRepository $fishbowlRepository,
```

Comment:
private readonly or readonly private? I saw you used `private readonly` on other places.

Reply:
It is not OK sorry ^^U I let phpstorm auto solve it. I will change the order in the `FishbowlService` file too.
`private readonly`